pytorch_pfn_extras/writing/_writer_base.py

```diff
@@ -89,11 +89,11 @@ def list(
             path_or_prefix: Optional[str] = None,
             recursive: bool = False,
     ) -> Iterator[str]:
+        path_or_prefix = self.get_actual_path(path_or_prefix)
```

Comment:
`get_actual_path` could be None here 😅

Reply:
yeah, I think this should fix it, `get_actual_path` fails and `os.scandir` admits `None`